src/govuk/components/pagination/template.test.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-env jest */
+
+const axe = require('../../../../lib/axe-helper')
+
+const { render, getExamples } = require('../../../../lib/jest-helpers')
+
+const examples = getExamples('pagination')
+
+describe('Pagination', () => {
```

Comment:
Would expect to see a few more tests that provide better coverage of the API of the component. It's probably worth having a read through the tests for a couple of other components to get an idea of the sort of things we'd tend to include.

Generally tests are better at covering the things that are not visually obvious in the review app. For example, ensuring that when users pass in additional classes they are applied correctly, text can be customised, text is sanitised and HTML is not, etc.

It might also make sense to add some tests to ensure that SVGs are `aria-hidden` / `focusable=false`; that rel=prev/next are applied correctly – anything that has semantic meaning or is important for accessibility but could easily be missed in a future change.